src/twitter/twitter-notifications/twitternotificationsyncadaptor.cpp

```diff
@@ -480,7 +480,9 @@ void TwitterNotificationSyncAdaptor::finishedFollowersHandler()
                 Notification *notification = createNotification(accountId, Follower);
                 notification->setItemCount(notification->itemCount() + newFollowers.size());
                 notification->setTimestamp(QDateTime::currentDateTimeUtc());
-                notification->setSummary(qtTrId("qtn_social_notifications_twitter_you")); // EE defined above
+                //: This label refers to the user of the device, eg "You" + "have 5 new followers on Twitter"
```

Comment:
Yes, that was just the context string, not the EE string.  But I've removed it from the context string now.